src/js/components/Section.js

```diff
@@ -2,12 +2,16 @@
 
 var React = require('react');
 var Box = require('./Box');
+var SkipLinkAnchor = require('./SkipLinkAnchor');
+var merge = require('lodash/object/merge');
 
 var CLASS_ROOT = "section";
 
 var Section = React.createClass({
 
-  propTypes: Box.propTypes,
+  propTypes: merge(Box.propTypes, {
+    primary: React.PropTypes.bool
```

Comment:
What is the context where we would have a primary Section without a primary Article?